hrms/hr/doctype/job_opening/job_opening.json

```diff
@@ -241,12 +242,19 @@
    "fieldtype": "Select",
    "label": "Salary Paid Per",
    "options": "Month\nYear"
+  },
+  {
+   "default": "1",
+   "depends_on": "publish",
+   "fieldname": "publish_applications_received",
+   "fieldtype": "Check",
+   "label": "Publish Applications Received"
```

Comment:
This label is a bit confusing for a user who has not seen the portal view. Maybe add a short description?